app/src/test/java/ingsw/group1/findmyphone/location/LocationMessageParserTest.java

```diff
@@ -0,0 +1,79 @@
+package ingsw.group1.findmyphone.location;
+
+import android.location.Location;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+/**
+ * Test for {@link LocationMessageParser}
+ *
+ * @author Giorgia Bortoletti
+ */
+
+@RunWith(MockitoJUnitRunner.class)
+public class LocationMessageParserTest {
+
+    private final double EX_LATITUDE = 110110110;
+    private final double EX_LONGITUDE = 151515151;
+
+    private String locationMessageRequest, locationMessageResponse;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use one line for each declaration, it enhances code readability.](https://app.codacy.com/manual/RicDeZen/ingsw-group1-findmyphone/pullRequest?prid=4929934)